data/human/news.txt

```diff
@@ -57,6 +57,9 @@ news "merchant anywhere"
 			"No bread is better than half a loaf, because then the passengers won't fight over it."
 			"A painting is worth a thousand words, because the profit margin on transporting visual art is much higher than on dry literature."
 			"A journey of a thousand miles has too short of a distance to make any profit."
+			"The other day I happened across a rock band whose transport flaked out. Good thing I had eight free seats for them, because it got me into a free concert at their next stop."
```

Comment:
```suggestion
			"The other day I happened across a rock band whose transport flaked out. Good thing I had seats for all eight of them, because it got me into a free concert at their next stop."
```
Bit more natural phrasing but also highlights specific number more directly.